src/components/Button/__stories__/Variant.stories.tsx

```diff
@@ -0,0 +1,32 @@
+import { ComponentStory } from '@storybook/react'
+import Button from '..'
+
+export const Variant: ComponentStory<typeof Button> = () => (
+  <>
+    <Button variant="primary">Primary</Button>
+    <Button variant="secondary">Secondary</Button>
+    <Button variant="success">Success</Button>
+    <Button variant="warning">Warning</Button>
+    <Button variant="info">Info</Button>
+
+    <Button variant="link">Link</Button>
+
+    <Button variant="primary-bordered">Primary bordered</Button>
+    <Button variant="secondary-bordered">Secondary bordered</Button>
+    <Button variant="success-bordered">Success bordered</Button>
+    <Button variant="warning-bordered">Warning bordered</Button>
+    <Button variant="info-bordered">Info bordered</Button>
+
+    <Button variant="primary-soft-bordered">Primary Soft bordered</Button>
+    <Button variant="success-soft-bordered">Success Soft bordered</Button>
+    <Button variant="warning-soft-bordered">Warning Soft bordered</Button>
+
+    <Button variant="transparent">Transparent</Button>
+  </>
+)
```

Comment:
Could we do this instead? For all stories

```suggestion
export const Variant: ComponentStory<typeof Button> = ({...props}) => (
  <>
    <Button variant="primary" {...props}>Primary</Button>
    <Button variant="secondary" {...props}>Secondary</Button>
    <Button variant="success" {...props}>Success</Button>
    <Button variant="warning" {...props}>Warning</Button>
    <Button variant="info" {...props}>Info</Button>

    <Button variant="link" {...props}>Link</Button>

    <Button variant="primary-bordered" {...props}>Primary bordered</Button>
    <Button variant="secondary-bordered" {...props}>Secondary bordered</Button>
    <Button variant="success-bordered" {...props}>Success bordered</Button>
    <Button variant="warning-bordered" {...props}>Warning bordered</Button>
    <Button variant="info-bordered" {...props}>Info bordered</Button>

    <Button variant="primary-soft-bordered" {...props}>Primary Soft bordered</Button>
    <Button variant="success-soft-bordered" {...props}>Success Soft bordered</Button>
    <Button variant="warning-soft-bordered" {...props}>Warning Soft bordered</Button>

    <Button variant="transparent" {...props}>Transparent</Button>
  </>
)
```

Doing this way will remove those:
![Screenshot 2022-11-29 at 09 56 55](https://user-images.githubusercontent.com/15812968/204484109-e0f65228-52b7-437f-abfa-4ce679597c55.png)